pkg/sql/execinfrapb/processors_changefeeds.proto

```diff
@@ -68,6 +68,15 @@ message ChangeFrontierSpec {
   // changefeed at that timestamp.
   repeated roachpb.Span tracked_spans = 1 [(gogoproto.nullable) = false];
 
+  // Checkpoint enables change aggregators to resume faster on restart.
+  message Checkpoint {
```

Comment:
Instead of making a new message, could you reuse the one in `pkg/jobs/jobspb/jobs.proto`?